vector/src/main/java/im/vector/app/core/extensions/TextView.kt

```diff
@@ -65,12 +75,39 @@ fun TextView.setTextWithColoredPart(@StringRes fullTextRes: Int,
     text = SpannableString(fullText)
             .apply {
                 setSpan(foregroundSpan, index, index + coloredPart.length, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+                if (onClick != null) {
+                    val clickableSpan = object : ClickableSpan() {
```

Comment:
Create a class `NoUnderlineClickableSpan` ?

Reply:
Not sure it's relevant